docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -703,6 +860,12 @@ $ oc exec openstackclient -- openstack network agent list
 +--------------------------------------+------------------------------+------------------------+-------------------+-------+-------+----------------------------+
 ----
 
+[NOTE]
+====
+After the data plane adoption completed, {OpenStackPreviousInstaller} cell controllers should be decomissioned.
```

Comment:
We should use fomulations akin to "after removing all services", to make it more clear that we don't mean just the main part of the adoption completed, but also after migrating off any leftover functionality like networkers, Swift, Ceph.

Reply:
We have agreed to address the docs review in follow up